content/_data/upgrades/2-361-1.adoc

```diff
@@ -0,0 +1,139 @@
+Make sure you upgrade all plugins through the *Plugin Manager* after upgrading Jenkins core.
+
+For more information about recent JVM metaspace changes, see link:https://www.youtube.com/watch?v=XqaQ-z70sQs[Taming Metaspace: a look at the machinery, and a proposal for a better one | FOSDEM 2020] and the following blog posts:
+
+* link:https://stuefe.de/posts/metaspace/what-is-metaspace/[What is Metaspace?]
+* link:https://stuefe.de/posts/metaspace/metaspace-architecture/[Metaspace Architecture]
+* link:https://stuefe.de/posts/metaspace/what-is-compressed-class-space/[What is Compressed Class Space?]
+* link:https://stuefe.de/posts/metaspace/sizing-metaspace/[Sizing Metaspace]
+* link:https://stuefe.de/posts/metaspace/analyze-metaspace-with-jcmd/[Analyze Metaspace with `jcmd VM.metaspace`]
+
+===== OpenJDK 11.0.16 and container awareness for cgroups v2
+
+OpenJDK 11.0.16 contains a fix for link:https://bugs.openjdk.org/browse/JDK-8230305[JDK-8230305], which is container awareness for link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[cgroups v2].
+
+Previously, there would be no container detection when running OpenJDK 11.0.15 on cgroups v2, and the limits from the container host would be used.
+As of OpenJDK 11.0.16, OpenJDK detects container limits through cgroups v2.
+For example, these limits affect the garbage collection (GC) algorithm selected by the JVM, the default size of the heap, the sizes of thread pools, and how default parallelism is determined for `ForkJoinPool`.
+
+For more information, refer to link:https://developers.redhat.com/articles/2022/04/19/java-17-whats-new-openjdks-container-awareness#tuning_defaults_for_containers[Java 17: What's new in OpenJDK's container awareness] and link:https://www.youtube.com/watch?v=ZmTUBKvSWzs[OpenJDK 17 | Container Detection Demo].
+
+
+==== Updated Jetty version
+
+The Jetty version has been updated from 9.4.46.v20220331 to 10.0.11.
+Support for OpenSSL-style PEM-encoded RSA private keys has been removed when running Jenkins with the embedded Jetty (Winstone) container and TLS.
+Specifically, the `--httpsPrivateKey` and `--httpsCertificate` flags have been removed in favor of the `--httpsKeyStore` flag.
+The removed flags had printed deprecation warnings since 2016, and were implemented with non-standard APIs that have since been removed from Java 17.
+The recommendation is migrating to the `--httpsKeyStore` flag, which takes a keystore as described in link:https://www.jenkins.io/doc/book/installing/initial-settings/#https-with-an-existing-certificate[the documentation].
+As of link:https://openjdk.org/jeps/229[JEP 229], link:https://en.wikipedia.org/wiki/PKCS_12[PKCS12] is the recommended keystore type.
+
+==== Updated minimum supported remoting version
+
+The minimum required remoting version has been increased to 4.2.1.
+Ensure that all agents are running a recent version of remoting prior to upgrading.
```

Comment:
Remoting is the name of a Jenkins sub-project, like "Stapler", so it should be capitalized. If we were referring to the general concept of a library that facilitates remote access (which is what the Jenkins project's Remoting is) then we could call it "a remoting library".